presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -182,7 +201,7 @@ public boolean isLimitGuaranteed(ConnectorSession session)
     @Override
     public Optional<JdbcTableHandle> getTableHandle(ConnectorSession session, SchemaTableName schemaTableName)
     {
-        TableHandleCacheKey key = new TableHandleCacheKey(JdbcIdentity.from(session), schemaTableName);
+        TableHandleCacheKey key = new TableHandleCacheKey(JdbcIdentity.from(session), getSessionProperties(session), schemaTableName);
```

Comment:
I believe this is redundant, but we should do https://github.com/prestosql/presto/pull/6161/commits/00d7b7b14789db46cf752fc933aa2a9670992f5e instead
("Add safeguards in table handles cache in JDBC")